modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-single-pvc-template-storage-pool_{context}"]
+= Creating a storage pool using a `pvcTemplate` in a host path provisioner (HPP) custom resource.
+
+If you have a single large persistent volume (PV) on your node, you might want to virtually divide the volume and use one partition to store only the HPP volumes. By defining a `storagePool` using a `pvcTemplate` in the HPP custom resource, you can virtually split the PV into multiple smaller volumes, providing more flexibility in data allocation.
```

Comment:
For references to actual object or field names (versus generic references), suggest adding whatever word is technically accurate after some of these items to qualify what it is. E.g.:

```
a `storagePool` array
```
```
a `pvcTemplate` storage pool
```

At least for the first instances per module, but preferably throughout. See also https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references.

Reply:
I tried to get everything and be as consistent as possible across all 6 files.